setup.py

```diff
@@ -79,6 +79,8 @@
                       'beautifulsoup4>=4.4.0',
                       'requests>=2.18.1',
                       'lxml>=4.2.1',
+                      'fastparquet>=0.4.1',
```

Comment:
remove fastparquet. Pyarrow alone should be sufficient (as the faster of the two)

Reply:
Agreed.